pkg/slacknotifier/slack.go

```diff
@@ -169,11 +212,32 @@ func SendEvent(eventType *testkube.WebhookEventType, execution testkube.Executio
 	if err != nil {
 		return err
 	}
-	if c != nil && c.SlackClient != nil {
-		_, _, err := c.SlackClient.PostMessage(c.ChannelId, slack.MsgOptionBlocks(view.Blocks.BlockSet...))
+
+	if slackClient != nil {
+		channels, _, err := slackClient.GetConversationsForUser(&slack.GetConversationsForUserParameters{})
 		if err != nil {
 			return err
 		}
+
+		if len(channels) > 0 {
+			channelID := channels[0].GroupConversation.ID
+
+			_, _, err := slackClient.PostMessage(channelID, slack.MsgOptionBlocks(view.Blocks.BlockSet...))
+			if err != nil {
+				return err
+			}
+		}
 	}
+
 	return nil
 }
+
+func getFailedStepsCount(steps []testkube.ExecutionStepResult) int {
```

Comment:
probably, it's better to move to testkube model package

Reply:
@vsukhin do you mean in model_execution.go?

Reply:
model_execution_result.go for example